bin/fuel-core/src/cli/run.rs

```diff
@@ -189,21 +189,49 @@ pub struct Command {
     #[arg(long = "native-executor-version", env)]
     pub native_executor_version: Option<StateTransitionBytecodeVersion>,
 
-    /// The starting gas price for the network
-    #[arg(long = "starting-gas-price", default_value = "0", env)]
+    /// The starting execution gas price for the network
+    #[arg(long = "starting-gas-price", default_value = "1000", env)]
     pub starting_gas_price: u64,
 
     /// The percentage change in gas price per block
-    #[arg(long = "gas-price-change-percent", default_value = "0", env)]
-    pub gas_price_change_percent: u64,
+    #[arg(long = "gas-price-change-percent", default_value = "10", env)]
+    pub gas_price_change_percent: u16,
 
     /// The minimum allowed gas price
     #[arg(long = "min-gas-price", default_value = "0", env)]
     pub min_gas_price: u64,
 
     /// The percentage threshold for gas price increase
     #[arg(long = "gas-price-threshold-percent", default_value = "50", env)]
-    pub gas_price_threshold_percent: u64,
+    pub gas_price_threshold_percent: u8,
```

Comment:
this is a breaking change

Reply:
Do we want to revert? It will only break if people used 256% or bigger.